lib/cocoapods/command/deintegrate.rb

```diff
@@ -1,5 +1,7 @@
 module Pod
   class Command
+    # @since 1.0.0.beta.1
```

Comment:
what about `@since Included in CocoaPods as of 1.0.0.beta.1` - or does `@since` require a semver?